### ISSUE TITLE
[Bug] Install-DbaFirstResponderKit installs the deprecated versions instead of the updated versions

### ISSUE THREAD

**Issue by elsimer** [sqlcollaborative/dbatools#6924]
### Report

When installing or updating FirstResponderKit from local file or from default latest version, the Deprecated folder in the zip file are included and overwrite updates.

#### Host used

- [X] powershell.exe
- [X] ISE
- [ ] VS Code
- [ ] Other (please specify)

#### Steps to Reproduce

` Install-DbaFirstResponderKit -SqlInstance localhost -Database master -Verbose`

#### Expected Behavior

Install should not include scripts listed in the Deprecated folder, which have _2005.sql appended to their filenames by default.

#### Actual Behavior

```
VERBOSE: Performing the operation "installing/updating sp_Blitz_SQL_Server_2005.sql in master." on target "localhost".
VERBOSE: Changed database context to 'master'.
SqlInstance  : localhost
InstanceName : MSSQLSERVER
ComputerName : VM-DWSQL01
Name         : sp_Blitz_SQL_Server_2005
Status       : Installed
Database     : master
```

### Environmental information

🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
Please provide the output of the below script

```PowerShell Version        : 5.1.18362.145
dbatools latest installed : 1.0.124
Culture of OS             : en-US
```

#### SQL Server:

```sql
Microsoft SQL Server 2016 (SP2-CU15) (KB4577775) - 13.0.5850.14 (X64)   Sep 17 2020 22:12:45   Copyright (c) Microsoft Corporation  Enterprise Edition: Core-based Licensing (64-bit) on Windows Server 2012 R2 Standard 6.3 <X64> (Build 9600: ) (Hypervisor) 
```

```sql
us_english
```

**Comment by andreasjordan** [comment]
Maybe you have an old local copy including those files (there was a bug some versions ago).
Could you rerun the command with `-Force` to update the local cache?

**Comment by jpomfret** [comment]
I can't replicate this on version 1.0.125
```
Install-DbaFirstResponderKit -SqlInstance mssql1 -Database master -Verbose
```
Then checking which stored procs got installed:
![image](https://user-images.githubusercontent.com/981370/97328945-6f6fd000-186e-11eb-8e35-cc4379961da3.png)

**Comment by elsimer** [comment]
> I can't replicate this on version 1.0.125
> 
> ```
> Install-DbaFirstResponderKit -SqlInstance mssql1 -Database master -Verbose
> ```
> 
> Then checking which stored procs got installed:
> ![image](https://user-images.githubusercontent.com/981370/97328945-6f6fd000-186e-11eb-8e35-cc4379961da3.png)

It could have been fixed (I ran it on 1.0.124) but the deprecated versions are also named sp_blitz, sp_blitzcache, and sp_blitzindex.

**Comment by elsimer** [comment]
When I ran it on 1.0.131, I no longer encounter the error.
